tools/ops/policylambda.py

```diff
@@ -77,13 +81,56 @@ def render(p):
                     'Properties': {
                         'Pattern': json.loads(e.render_event_pattern())}
             }
+        properties['Events'] = revents
+
+    elif p.execution_mode == 'config-rule':
+        properties.pop('Delay', None)
 
-    properties['Events'] = revents
     return {
         'Type': 'AWS::Serverless::Function',
         'Properties': properties}
 
 
+def render_config_rule(p):
+    policy_lambda = mu.PolicyLambda(p)
+    policy_lambda.arn = get_lambda_arn(policy_lambda.name)
+    config_rule = policy_lambda.get_events(Session)
+
+    properties = config_rule[0].get_rule_params(policy_lambda)
+
+    exec_mode_type = p.data.get('mode', {'type': 'pull'}).get('type')
+    if exec_mode_type == 'config-poll-rule':
+        properties.pop('Scope', None)
+
+    attributes = {}
+    attributes['Type'] = 'AWS::Config::ConfigRule'
+    attributes['DependsOn'] = resource_name(p.name) + "Invoke"
+    attributes['Properties'] = properties
+    return attributes
+
+
+def render_invoke(name):
+    return {
+        "DependsOn": name,
+        "Type": "AWS::Lambda::Permission",
+        "Properties": {
+            "Action": "lambda:InvokeFunction",
+            "FunctionName": {"Ref": name},
+            "Principal": "config.amazonaws.com"
+        }
+    }
+
+
+def get_lambda_arn(function_name):
+    sts = boto3.client("sts")
```

Comment:
we don't really want the arn, we want a cfn reference to the function. we should update mu.py here to support pass through on the reference.